src/Worker.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace Equip\Queue;
+
+use Equip\Queue\Driver\DriverInterface;
+use Exception;
+use League\Event\EmitterInterface;
+
+class Worker
```

Comment:
This class is lacking the most, needs to add options such as stop on exception, max run time, etc.

It currently is in working form.